apps/openmw/mwclass/mobile.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef GAME_MWCLASS_MOBILE_H
+#define GAME_MWCLASS_MOBILE_H
+
+#include "../mwworld/class.hpp"
+
+namespace ESM
+{
+    struct GameSetting;
+}
+
+namespace MWClass
+{
+    /// \brief Class holding functionality common to Creature and NPC
+    class Mobile : public MWWorld::Class
+    {
+    protected:
+
+        Mobile();
+
+    public:
+        ~Mobile();
```

Comment:
Not going to happen since the entire point of the Class hierarchy is to access the sub-classes via the Class interface. But we are getting into nitpicking area here.